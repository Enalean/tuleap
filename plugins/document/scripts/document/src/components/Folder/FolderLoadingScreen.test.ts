/*
 * Copyright (c) Enalean 2021 -  Present. All Rights Reserved.
 *
 *  This file is a part of Tuleap.
 *
 * Tuleap is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
 *
 * Tuleap is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Tuleap. If not, see <http://www.gnu.org/licenses/>.
 *
 */

import { shallowMount } from "@vue/test-utils";
import FolderLoadingScreen from "./FolderLoadingScreen.vue";

describe("FolderLoadingScreen", () => {
    it("displays the folder loading screen", () => {
        const wrapper = shallowMount(FolderLoadingScreen, {});

        expect(wrapper.element).toMatchSnapshot();
    });
});
