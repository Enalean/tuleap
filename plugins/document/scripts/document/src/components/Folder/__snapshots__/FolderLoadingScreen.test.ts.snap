// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderLoadingScreen displays the folder loading screen 1`] = `
<div
  class="tlp-card"
>
  <skeleton-table-stub
    cell_classes=",document-tree-cell-owner,document-tree-cell-updatedate"
    icons="fa-solid fa-folder,fa-solid fa-folder,fa-regular fa-file,fa-regular fa-file,fa-regular fa-file"
    nb_cols="3"
    nb_rows="5"
  />
</div>
`;
